src/main/cli/cli.c

```diff
@@ -5775,51 +5720,11 @@ static void printConfig(char *cmdline, bool doDiff)
             dumpAllValues(HARDWARE_VALUE, dumpMask, "master");
         } else {
             dumpAllValues(MASTER_VALUE, dumpMask, "master");
-
-            if (dumpMask & DUMP_ALL) {
```

Comment:
All of this seems to have been lost.

Reply:
Its the start of removing the profile code.